### PR TITLE
Microverse Project 1: Basic HTML Page Structure

### PR DESCRIPTION
**Pull request by Janluc**
## Basic HTML Page Structure with Video and Image Embedding

This project consists of a Youtube Video player clone made using
- HTML
- CSS

In this project we Embedded images and a video to emulate the youtube layout. 
We also have extensive use of flexbox in order to properly align all the elements to best emulate the intended layout.
The page includes a navbar with a Search Bar and various icons, as well as a footer that would lead to more information regarding the company (YouTube)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+                    </span>
+                </li>
+
+
+            </ul>
+        </div>
+    </nav>
+
+
+    <!-- ============ Video Section =========-->
```
Comment (teekaytech) [inline index.html]: You may want to remove this comment, although this is a personal opinion.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+            <div id="vid-iframe">
+                <iframe width="800" height="500" src="https://www.youtube.com/embed/IUMTaAQ43lY" frameborder="0"
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                    allowfullscreen>
+                </iframe>
+            </div>
+            <section class="container" id="video-player">
+
+
+                <!-- ============ Video Description =========== -->
```
Comment (teekaytech) [inline index.html]: ...and this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+                </p>
+                <br>
+                <button id="show-btn">SHOW MORE</button>
+            </section>
+
+            <br>
+
+
+
+            <!--======================Comment section =========================  -->
```
Comment (teekaytech) [inline index.html]: ...and this

File: index.html [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+            <div class="loading">
+                <p><i class="fas fa-spinner "></i></p>
+            </div>
+        </div>
+
+
+        <!-- ================ Video Side Bar ================== -->
```
Comment (teekaytech) [inline index.html]: ...and this

File: index.html [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+                        </span>
+                    </li>
+                </ul>
+
+                <button id="more-videos">SHOW MORE</button>
+            </section>
+        </div>
+
+
+        <!--=================== Footer =================-->
```
Comment (teekaytech) [inline index.html]: ...and this

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+    font-size: 18px;
+}
+
+.search-btn {
+    margin-left: 0;
+    border: 1px solid grey;
+    border-left: 0;
+    width: 10%;
+
+
```
Comment (teekaytech) [inline styles/styles.css]: This white space is not necessary. Kindly remove it.

File: styles/styles.css [hunk 7]
```diff
@@ -0,0 +1,357 @@
...
+    font-size: 16px;
+    font-weight: 600;
+}
+
+
+
+
+
+
+
```
Comment (teekaytech) [inline styles/styles.css]: Kindly remove this too.

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Changes Required :red_circle: 

**Hi, @Janluc & @sumon0002001 :wave:. Congratulations on submitting your first pull request successfully. You guys have done a great job here. However, there are a couple of changes to be made.**

**[ GENERAL]**

- You committed a file not related to the project. You can use .gitignore to exclude such a file from being committed or delete the file completely if it's not relevant to the project.
- Check the selected section of your code below for other changes.


**[OPTIONAL]**
- There are some white spaces in your index.html file. Try to remove some that are not necessary.
- Commenting about codes is fine. However, it can sometimes mess up your code. Besides, those comments you made in your index.html file are unnecessary.
- Try to add the comments section to the project.

**COMMENT(S)**
- Be careful with the way you use CSS selectors. Although there are different opinions about this, I will advise you to get used to using class names for CSS declarations.

### Overall, this is a very nice implementation. Kindly fix the required changes and submit another code review request form once you're done.

**I'm available on slack (@taofeekolalere) or GitHub (@teekaytech)**
Happy Coding:man_technologist:

**Review by javitocor — APPROVED** [review]
#PROJECT APPROVED 🥇 

Good job with your first project and with the changes made! Congratulations! 🥳 

Remember to fill up the completion form and move into the next project ⏩ 

Reminders:
- Remember to set up linters in the next project, is mandatory 🧪 
- Add a live link in your README file would be nice, you can use https://raw.githack.com/ 👍 

Good Luck, team 😃

**Pull request merged**